webapp/components/post_view/components/post_message_view.jsx

```diff
@@ -55,6 +58,10 @@ export default class PostMessageView extends React.Component {
             return true;
         }
 
+        if (!Utils.areObjectsEqual(nextProps.liteUsernameMap, this.props.liteUsernameMap)) {
+            return true;
+        }
```

Comment:
Noted, I thought about that performance issue as well initially. However, it seems not smooth when the profile popover of both username and picture in user's post get updated immediately (as soon as there's update in user's profile) while the at-mention in message is not. 
But I'm all in with performance and will update this PR accordingly.